mayastor/src/bdev/dev/nvmx/controller.rs

```diff
@@ -60,25 +74,26 @@ impl<'a> NvmeControllerInner<'a> {
             ctrlr,
             adminq_poller,
             namespaces: Vec::new(),
+            io_device,
         }
     }
 }
 
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Copy, Clone)]
 pub enum NvmeControllerState {
     Initializing,
     Running,
-    Resetting,
-    Destroying,
+    Unconfiguring,
+    Unconfigured,
 }
 
 impl ToString for NvmeControllerState {
     fn to_string(&self) -> String {
         match *self {
             NvmeControllerState::Initializing => "Initializing",
             NvmeControllerState::Running => "Running",
-            NvmeControllerState::Resetting => "Resetting",
-            NvmeControllerState::Destroying => "Destroying",
+            NvmeControllerState::Unconfiguring => "Unconfiguring",
+            NvmeControllerState::Unconfigured => "Unconfigured",
         }
         .parse()
```

Comment:
I think you can call `"Unconfigured".to_string()` and avoid the unwrap here.

Reply:
Good point, thanks.